tests/lib.rs

```diff
@@ -0,0 +1,16 @@
+extern crate libftd2xx_ffi;
```

Comment:
```suggestion
```

`extern crate` is not needed anymore: https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html